pkg/cmd/root/help.go

```diff
@@ -25,7 +25,7 @@ func rootUsageFunc(command *cobra.Command) error {
 		return nil
 	}
 
-	flagUsages := command.LocalFlags().FlagUsages()
+	flagUsages := command.Flags().FlagUsages()
```

Comment:
@mislav @vilmibm Is there a reason why this was currently only using `LocalFlags()`?